topics/galaxy-project/tutorials/outputs/tutorial.md

```diff
@@ -0,0 +1,125 @@
+---
+layout: tutorial_hands_on
+
+title: "4. Galaxy project outputs"
+questions:
+  - "What are the main identifiable project outputs?"
+  - "How can Galaxy offer 'free' analysis computational resources?"
+  - "How does the open source community support Galaxy"
+  - "How are new project outputs or resources created?"
+objectives:
+  - "Understand Galaxy outputs and open science services"
+  - "Understand how different communities work together to make things happen"
+  - "Understand opportunities for engagement and contributing your skills"
+ 
+time_estimation: 10m
+key_points:
+  - "The Galaxy project creates many highly valued open science resources"
+  - "Unlike most open source projects, Galaxy offers important services and community activities in addition to source code"
+  - "Participants engage with, and add value to the project outputs on their own individual terms"
+ 
+contributors:
+  - ggsc
+  - fubar2
```

Comment:
If you like can use our fancy new contributions syntax to more accurately record contributions to a training material, declaring which people should be recorded for authorship or just editing.

```
contributions:
  authorship:
    - shiltemann
  editing:
    - bebatut
    - hexylena
  funding:
    - carpentries
  testing:
    - userX
  ux:
    - userY
  infrastructure:
    - userZ
```

see the [full gtn tutorial schema documentation](https://training.galaxyproject.org/training-material/topics/contributing/tutorials/schemas/tutorial.html) for details on the individual roles.